File: front/src/app/shared/models/BebidaDTO.ts
export class BebidaDTO {
    id : number = 0;
    nome: string = '';
    tamanhoValor: BebidaTamanhoValorDTO[] =[];
}
export class BebidaTamanhoValorDTO {
    tamanho: string = '';
    valor: number = 0;
}

File: back/src/main/java/br/com/nostramassa/gestao/dtos/pedido/BebidaDTO.java
package br.com.nostramassa.gestao.dtos.pedido;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class BebidaDTO {
	
	private Long id;

	private String nome;
	
	private List<BebidaTamanhoValorDTO> tamanhoValor;


}